src/dispatch/incident/service.py

```diff
@@ -196,35 +207,15 @@ def create(
     )
 
     # We add the reporter to the incident
-    reporter_participant = participant_flows.add_participant(
+    participant_flows.add_participant(
         reporter_email, incident.id, db_session, ParticipantRoleType.reporter
     )
 
-    incident_commander_email = resolve_incident_commander_email(
-        db_session,
-        reporter_email,
-        incident_type.name,
-        "",
-        title,
-        description,
-        incident_priority.page_commander,
+    # Add other incident roles (e.g. commander and liaison)
+    assign_incident_role(
+        db_session, incident, reporter_email, ParticipantRoleType.incident_commander
     )
```

Comment:
nit: we could avoid passing `reporting_email` and use the incident `reporter` hybrid property inside `assign_incident_role()`